metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -48,6 +48,7 @@
     "s3_sensor_with_formatter_flow.py",
     "validate_s3_sensor_flows.py",
     "toleration_and_affinity_flow.py",
+    "flow_triggering_flow.py",  # TODO(yunw): This flow is a manual test - automate before merging
```

Comment:
what's remaining to automate? Does this needs a JIRA task?

Reply:
Will use the same Jira ticket. Shouldn't be too complicated. Work involved are:
- Publish "flow B" as a pipeline before triggering the test
- Resolve the `KFP` not found issue as it adds around 3 min (or more if I add more complete test suite) by requiring each step to install the dependencies.

I will use the same ticket since leaving some test incomplete doesn't sound like a completed Jira task.

Reply:
Flow B is now automatically published and deleted per test run. Import issue above is also resolved.

The only remaining item is to correctly set namespace - will do after demo on 1/25